frontend/src/components/LoginForm/LoginForm.jsx

```diff
@@ -7,12 +7,15 @@ import { useNavigate } from "react-router-dom";
 export default function LoginForm({ setUser }) {
     const [username, setUsername] = useState("");
     const [password, setPassword] = useState("");
+    const [error, setError] = useState("");
 
     const navigate = useNavigate();
 
     async function handleLogin(event) {
         event.preventDefault();
 
+        setError("");
```

Comment:
Maybe a more accurate way of representing the error state is that error is either NULL or a string

Reply:
I'm making an error component and it will show on the next pull request so let me know what to do once you see the updated code.